addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRule.java

```diff
@@ -71,7 +72,14 @@ public class HiddenFilesScanRule extends AbstractHostPlugin {
                     CommonAlertTag.WSTG_V42_CONF_05_ENUMERATE_INFRASTRUCTURE);
     static final String PAYLOADS_FILE_PATH = "json/hidden_files.json";
 
-    private static final List<String> HIDDEN_FILES = new ArrayList<>();
+    private static final List<String> HIDDEN_FILES =
+            Arrays.asList(
+                    // Source Control paths to look for, per:
+                    // https://twitter.com/intigriti/status/1533050946212839424
+                    ".hg", // Mercurial
+                    ".bzr", // Bazaar
+                    "._darcs", // Darcs
+                    "BitKeeper");
```

Comment:
Why not add these to `addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json` with the other payloads?

Reply:
Sorry, just saw your comment.